examples/2d/bloom_2d.rs

```diff
@@ -110,6 +139,7 @@ fn update_bloom_settings(
                 bloom.prefilter.threshold_softness
             ));
             text.push_str(&format!("(I/K) Horizontal Scale: {}\n", bloom.scale.x));
+            text.push_str(&format!("(O) Tonemapper: {:?}\n", tonemapper.tonemapper));
```

Comment:
It would be nice if these instructions were still displayed, and the cycling still worked when bloom is disabled.